pkg/morph/event/netmap/update_peer.go

```diff
@@ -46,7 +47,7 @@ func (s UpdatePeer) NotaryRequest() *payload.P2PNotaryRequest {
 func (s *UpdatePeer) decodeState(state int64) error {
 	switch s.state = nodestate.Type(state); s.state {
 	default:
-		return fmt.Errorf("unsupported node state %d", state)
+		return fmt.Errorf("%w: unsupported node state %d", errors.ErrUnsupported, state)
```

Comment:
Ditto

Reply:
thought about it, the full version is "unsupported operation: unsupported node state %d" now. cannot say i would be sad if i saw it, it may fit my expectation about error chain construction logic but don't mind dropping